app/views/content/non-uk-teachers/fees-and-funding-for-non-uk-trainees.md

```diff
@@ -111,23 +111,12 @@ You will also need a [visa or immigration status](/non-uk-teachers/visas-for-non
 
 ## Financial support for non-UK trainee languages and physics teachers
```

Comment:
@laurahermionetennant92 do we still need this separate section as it's now only bursaries and scholarships? So would make sense to incorporate any extra info into the bursaries and scholarships section below?

Reply:
yes good point